### PR TITLE
feature: py-invoke: new python package

### PR DESCRIPTION
**Pull request by goxberry**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-invoke/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+class PyInvoke(PythonPackage):
+    """Pythonic task execution"""
+
+    homepage = "http://www.pyinvoke.org/"
+    url      = "https://pypi.io/packages/source/i/invoke/invoke-1.2.0.tar.gz"
+
+    version('1.2.0', 'dc492f8f17a0746e92081aec3f86ae0b4750bf41607ea2ad87e5a7b5705121b7')
+
+    extends('python')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-invoke/package.py]: This line isn't needed, it's already present in the `PythonPackage` base class.

File: var/spack/repos/builtin/packages/py-invoke/package.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    depends_on('python@2.7:2.8,3.4:')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-invoke/package.py]: `type=('build', 'run')`

File: var/spack/repos/builtin/packages/py-invoke/package.py (hunk 1 above)
Comment (goxberry) [reply var/spack/repos/builtin/packages/py-invoke/package.py]: Fixed in [073ca50](https://github.com/spack/spack/commit/073ca504514c22d79e6ae03a12084b7fa47b9bca).

File: var/spack/repos/builtin/packages/py-invoke/package.py (hunk 2 above)
Comment (goxberry) [reply var/spack/repos/builtin/packages/py-invoke/package.py]: Also fixed in [073ca50](https://github.com/spack/spack/commit/073ca504514c22d79e6ae03a12084b7fa47b9bca).

**Comment by goxberry** [comment]
@adamjstewart Should be ready for another review.

**Pull request merged**